DOC-1725: Edit RC Quick Start to reflect new workflow (#2199)
RedisLabs/redislabs-docs: content/rc/rc-quickstart.md

You'll learn how to:

> 1.  Create an account with a free subscription and database

Reviewer: ```suggestion
1.  Create an account, a free subscription, and a database
```

Reviewer: Even though these are are all done together in one process, it might be good to word it so people understand that they can be done separately.

1.  Connect to your database

If you already have an account, see [Create a Fixed subscription]({{< relref "/rc/subscriptions/create-fixed-subscription" >}}) to create a Free 30MB subscription. Free plans are a tier of fixed plans; this provides an easy upgrade path when you need it.

If you already have a subscription, see [Manage subscriptions]({{< relref "/rc/subscriptions/" >}}) and [Manage databases]({{< relref "/rc/databases/" >}}).

## Create an account

To create a new account with a free subscription and database:

1. On the Redis Cloud [admin console](https://app.redislabs.com/#/login), select [Sign up](https://redis.com/try-free/).

1. Enter your information in the form and select **Get Started**, or sign up with Google or Github.

> 1. When you receive the activation email, select **Activate account** to be taken to the Redis Cloud [admin console](https://app.redislabs.com/#/login).

Reviewer: ```suggestion
1. In the activation email, select **Activate account** to go to the Redis Cloud [admin console](https://app.redislabs.com/#/login).
```

    {{<image filename="images/rc/quickstart-quick-dialog.png" width="75%" alt="Dialog to create your free subscription." >}}{{< /image >}}

> 1. Select your preferred cloud vendor and region from the dialog.

Reviewer: ```suggestion
1. Select your preferred cloud vendor and region.
```

1. Select **Let's start free** to create your subscription and database.

>     If you would rather customize your subscription and database, Select **Create a custom database** to be sent to the Add subscription page. From there, you can [create a fixed subscription]({{< relref "/rc/subscriptions/create-fixed-subscription" >}}) or [create a flexible subscription]({{< relref "/rc/subscriptions/create-fixed-subscription" >}}).

Reviewer: ```suggestion
    If you would rather customize your subscription and database, select **Create a custom database** to go to the **Add subscription** page. From there, you can [create a fixed subscription]({{< relref "/rc/subscriptions/create-fixed-subscription" >}}) or [create a flexible subscription]({{< relref "/rc/subscriptions/create-fixed-subscription" >}}).
```

> 1.  You're taken to the **Overview tab** for your new subscription.

Reviewer: This is more of a result of the previous step than an action step in itself.

    {{<image filename="images/rc/quickstart-subscription-overview.png" width="75%" alt="Overview tab showing your new subscription and database." >}}{{< /image >}}

>     Select the database name to view the **Configuration** tab for your new database.

Reviewer: ```suggestion
1. From the **Overview** page, select the database name to view the **Configuration** page for your new database.
```

> Once you have the connection details, you can connect in a variety of ways, including:

Reviewer: I would put a heading here to point out that we are now connecting to our database.

> - Using the `redis-cli` utility

Reviewer: what commands do they need to use with the redis cli? Usually we don't duplicate but in a quick start you want all the info they need at their fingertips.
